.github/workflows/deploy.yml

```diff
@@ -11,22 +11,22 @@ on:
 
 jobs:
   version_bump:
-    name: "Bump version"
+    name: 'Bump version'
```

Comment:
Not sure we even need this action anymore? I dont think we are going to try to automate the version numbers at this point, given the multiplicity of apple orgs and build flavors.